polaris-tokens/src/themes/base/color.ts

```diff
@@ -221,7 +222,8 @@ export type ColorTextAlias =
   | 'avatar-text-on-bg-fill'
   | 'avatar-three-text-on-bg-fill'
   | 'avatar-two-text-on-bg-fill'
-  | 'video-thumbnail-play-button-text-on-bg-fill';
+  | 'video-thumbnail-play-button-text-on-bg-fill'
+  | 'badge-new-text';
```

Comment:
Not blocking but I noticed that this file appears to be in alpha order. If we want to preserve that I think this new values should be listed after `avatar-two-text-on-bg-fill`.